platform/stm32f4xx/export/platform/spi_bus.hpp

```diff
@@ -415,9 +450,7 @@ ecl::err spi_i2s_bus< spi_i2s_config >::do_xfer()
     //    while (SPI_I2S_GetFlagStatus(spi, SPI_FLAG_BSY) == SET)
```

Comment:
Oh, this is an old rudiment. Could you please remove it if you consider it unnecessary here?